config/configmaps/install-log-regexes-configmap.yaml

```diff
@@ -149,6 +149,24 @@ data:
       - "The vpc ID .* does not exist"
       installFailingReason: AWSVPCDoesNotExist
       installFailingMessage: The AWS VPC does not exist
+    - name: VPCDhcpOptionAssociationAbsent
+    # https://bugzilla.redhat.com/show_bug.cgi?id=2032521
+      searchRegexStrings:
+      - "error .* applying changes to module.vpc.aws_vpc_dhcp_options_association"
+      installFailingReason: VPCDhcpOptionAssociationAbsent
+      installFailingMessage: The VPC Dhcp Option Association absent after creation
+    - name: TargetGroupNotFound
+    # https://bugzilla.redhat.com/show_bug.cgi?id=1898265
+      searchRegexStrings:
+      - "TargetGroupNotFound"
+      installFailingReason: TargetGroupNotFound
+      installFailingMessage: Target Group cannot be found after creation
+    - name: RouteTablePrivateRoutesAbsent
+    # https://bugzilla.redhat.com/show_bug.cgi?id=2033256
+      searchRegexStrings:
+      - "ERROR When applying changes to module.vpc.aws_route_table.private_routes"
```

Comment:
Same comment here about adding the "was present, but now absent" piece to the regex.